Src/DBProj/DBProj.sqlproj

```diff
@@ -68,11 +68,15 @@
     <Build Include="Tables\Products.sql" />
     <Build Include="Tables\Users.sql" />
     <Build Include="Procedures\dbo_SaveUser.sql" />
+    <Build Include="Tables\ProductCategory.sql" />
```

Comment:
ProductCategory.sql file is missing in commit, it seems it was not marked added while commiting through tortoisegit

Reply:
ok I do it again.